_documentation/05-User-Manual.md

```diff
@@ -161,12 +167,16 @@ ___
 
     ![Webp net-resizeimage (14)](https://user-images.githubusercontent.com/43112419/97453631-ef279880-195b-11eb-819a-e24f2c7820ee.png)
 
+    The option can be used to configure networking settings such as the hostname and also see the list of interfaces/addresses. 
```

Comment:
The option can be used to configure networking settings such as the hostname and also **to** see the list of interfaces/addresses.